grants/accepted_grant_applications.md

```diff
@@ -1,6 +1,8 @@
-# Accepted Grant Applications <!-- omit in toc -->
+# Accepted Grant Applications [WIP!] <!-- omit in toc -->
 
-This page gives an overview of accepted grants and a link to their GitHubs. Keep in mind that not all of the accepted grants have already delivered software.
+> **:loudspeaker:** This page provides an overview of accepted grant applications, their progress and a link to their GitHub repositories. In cases where the link points to an organization, you should be aware that the grant application itself **is often an independent project unrelated to other work done by the teams**. Besides, **there is a clear difference between an application being accepted and the successful delivery of the respective project**, and only teams that have successfully delivered a milestone are allowed to make public announcements on the matter or to use our [badge](https://github.com/w3f/General-Grants-Program/blob/master/grants/grant-badge-guidelines.md). The badge can also never be used as a general endorsement for a team. Violations to this policy can be reported [here](mailto:grants@web3.foundation).
+
+> Furthermore, the page lists terminations which happened due to a breach of the terms of the grants programs. Additionally, teams might have decided to stop working on the grant (though not necessarily on the project itself) for various reasons, which are not reflected on this sheet.   
```

Comment:
The thought process behind this was that it's not only the reasons that are not reflected on the sheet, but even whether teams have stopped working on a grant or not.

```suggestion
> Furthermore, the page lists terminations which happened due to a breach of the terms of the grants programs. Additionally, teams might have decided to stop working on the grant—though not necessarily on the project itself—for various reasons, which is not reflected on this sheet.

> Besides, **there is a clear difference between an application being accepted and the successful delivery of the respective project**, and only teams that have successfully delivered a milestone are allowed to make public announcements on the matter or to use our [badge](https://github.com/w3f/General-Grants-Program/blob/master/grants/grant-badge-guidelines.md). The badge can also never be used as a general endorsement for a team. Violations to this policy can be reported [here](mailto:grants@web3.foundation).
```